### DISCUSSION TITLE
Native Image Committer and Community Meeting 2022-09-22

### DISCUSSION THREAD

**Discussion by christianwimmer** [oracle/graal#5046] [Show and tell]
List of all past and upcoming meetings: #3933

## New and Noteworthy

* Feature freeze for GraalVM 22.3 has passed. Bugfixes are still accepted for a few weeks.

* JDK 19 / Loom support
  #5012 Stack allocated memory must not be used in virtual threads because the memory location switches when the physical carrier thread of the virtual thread changes.

* Native Image API
  #4989 The JDK initializes all annotation classes even when looking up a single annotation, this API provides a workaround for that.
  #4945 Add a resource programatically that does not exist in the hosting image builder VM.
  44e68777b130c8ee781c72b0e24dfec922712d85 Deprecate @AutomaticFeature. Please register all features explicitly in a native-image.properties file
     
* Performance
  #4929
  
* Monitoring / Tools
  #4811
  #5039 Work towards JFR sampling based profiling
  
## Deep Dive

Standalone points-to analysis contributed by Alibaba

## Possible deep dive topics for next meeting

Please send suggestions in the GraalVM Slack channel

**Comment by ziyilin** [comment]
My PPT for today's talk is here:
[Standalone Pointsto Analysis - community talk.pptx](https://github.com/oracle/graal/files/9626932/Standalone.Pointsto.Analysis.-.community.talk.pptx)
